javascript.md

```diff
@@ -71,18 +94,37 @@ Object.keys(person).forEach(lang.hitch(this, function(prop){
 });
 ```
 
+## Tools and configuration
+
+### Generic
+
+- [Jshint file][jshint-file]
+
+### Sublime Text
+
+- Package Control [installation][package-control-install]
+- Basic [Sublime Text User settings][sublime-settings]
+- Sublime Text [Package Control settings][sublime-package-control] with some nice packages to have.
+
 ### ChangeLog
 
 #### 2015.05.08
 
 - Initial Draft
 
+- Add more general tips.
+
 <!-- links -->
 
+[google-v8-optimizations]:https://developers.google.com/v8/design
+[memory-management-keynote]:https://speakerdeck.com/addyosmani/javascript-memory-management-masterclass
+[memory-management-video]:https://www.youtube.com/watch?v=LaxbdIyBkL0
 [steve-kwan-best-practices]:https://github.com/stevekwan/best-practices/blob/master/javascript/best-practices.md
-
 [javascript-the-good-parts]:http://javascript.crockford.com/
-
 [learning-javascript-design-patterns]:http://addyosmani.com/resources/essentialjsdesignpatterns/book/
-
 [jsdoc3]:https://github.com/jsdoc3/jsdoc
+
+[jshint-file]:{{ site.baseurl }}{{ post.url }}/files/jshintrc
```

Comment:
:( I viewed the file here, which is where I'd expect most users to view it, and it's not working for me.